pkgs/applications/misc/slstatus/default.nix

```diff
@@ -1,20 +1,22 @@
-{ stdenv, fetchgit, pkgconfig, writeText, libX11, conf ? null }:
+{ stdenv, fetchgit, pkgconfig, writeText, libX11, conf ? null, patches ? [] }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   pname = "slstatus";
-  version = "unstable-2018-04-16";
+  version = "unstable-2019-02-17";
```

Comment:
If I'm not mistaking, [`b14e039639ed28005fbb8bddeb5b5fa0c93475ac`][commit] was authored on 16 Feb 2019.

[commit]:https://git.suckless.org/slstatus/commit/b14e039639ed28005fbb8bddeb5b5fa0c93475ac.html

Reply:
That's odd, `nix-prefetch-git` told me it's from the 17th. Looks like you're right though, I'll change it to the 16th :+1: